### DISCUSSION TITLE
Missing Hover Styles for Post Title, Product Title, Price Widgets, etc. Widgets

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#22684] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

In the Elementor repository, there is an issue regarding the absence of hover styles for certain widgets, specifically the post title, product title, and price widgets. There is no option to set style for hover.

### Describe the solution you'd like

Add option to style these for hover 

### Describe alternatives you've considered

I use custom CSS, which is not really ee=ffect, also I can't use elementor global variable colors in it.

**Comment by ghost** [comment]
+1

**Comment by pumadog** [comment]
This could be solved with typo controls for containers: https://github.com/orgs/elementor/discussions/20529
